api/controllers/shabads.js

```diff
@@ -273,7 +273,7 @@ exports.angs = async (req, res) => {
   const sinceDate = req.query.updatedsince ? lib.isValidDatetime(req.query.updatedsince) : null;
 
   if (!lib.isRangeOfNumbers(PageNo)) {
```

Comment:
I was initially going to add a sanity check like if(typeof PageNo !== "string"), but it turns out the isRangeOfNumbers function already does that: https://github.com/KhalisFoundation/banidb-api/blob/fix-pageno-edgecases/api/lib/index.js#L35

this function would return false in the case PageNo isn't a number and thus default the value (now) to '1'